include/util/static_rtree.hpp

```diff
@@ -43,16 +43,19 @@ namespace util
 template <class EdgeDataT,
           class CoordinateListT = std::vector<Coordinate>,
           bool UseSharedMemory = false,
-          std::uint32_t BRANCHING_FACTOR = 128,
-          std::uint32_t LEAF_NODE_SIZE = 128,
-          std::uint32_t MAX_NUM_CANDIDATES = LEAF_NODE_SIZE * 10>
+          std::uint32_t BRANCHING_FACTOR = 64,
```

Comment:
Can you share why you changed the `BRANCHING_FACTOR` back to `64`? If we lower the node size it will double the  number of leaves which will create another level in the RTree doubling memory consumption again. Increase the `BRANCHING_FACTOR` keeps the hierarchy small and overhead smaller.